packages/gatsby-plugin-vanilla-extract/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gatsby-plugin-vanilla-extract",
+  "version": "2.0.0",
```

Comment:
@moonmeister Needs to chime in here but I think the correct way of doing this would be use the old current latest version (so `1.0.6`) and then adding a changeset that has the major change in it.

Also @KyleAMathews you'll need to connect with @moonmeister regarding npm credentials

Reply:
Correct. Leave the current version and you can add a changeset to bump.

Reply:
@moonmeister Is it okay to run the `cs` script for this?

I ran it and the changes in the [last commit](https://github.com/gatsby-uc/plugins/pull/148/commits/907876dfbb49d2d7e8d2db743c5455f1a7569750) got added

Reply:
yup. looks good.